### PR TITLE
Add MeshJS contract & vodka

### PR DESCRIPTION
**Pull request by HinsonSIDAN**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,13 +56,19 @@
...
 - [Gift Card NextJS](https://github.com/adalicious/aiken-gift-card) - A similar gift card example but with NextJS
+- [Gift Card MeshJS](https://meshjs.dev/smart-contracts/giftcard) - Gift card contract with full integration with MeshJS, offering code example and explanation.
+- [Escrow MeshJS](https://meshjs.dev/smart-contracts/escrow) - Escrow contract with full integration with MeshJS, offering code example and explanation.
+- [Marketplace MeshJS](https://meshjs.dev/smart-contracts/marketplace) - Cardano native token marketplace contract with full integration with MeshJS, offering code example and explanation.
+- [Payment Splitter MeshJS](https://meshjs.dev/smart-contracts/payment-splitter) - Payment splitter contract with full integration with MeshJS, offering code example and explanation.
+- [Simple Swap MeshJS](https://meshjs.dev/smart-contracts/swap) - Simple swap contract for token exchange with full integration with MeshJS, offering code example and explanation.
+- [Vesting Contract MeshJS](https://meshjs.dev/smart-contracts/vesting) - Vesting contract for token exchange with full integration with MeshJS, offering code example and explanation.
```
Comment (KtorZ) [inline README.md]: Thanks for the submission here, but I don't think that the awesome list is the best place for those. Reason being that none of these tutorials actually contain Aiken code. Rather, they focus on the off-chain part. It seems like they would be much more appropriate on the website (https://github.com/aiken-lang/site) as "off-chain" tutorials, presenting alternative to Lucid.

Comment (HinsonSIDAN) [reply README.md]: have replaced link to aiken workspace instead, as these collection of scripts has been useful for many

File: README.md [hunk 2]
```diff
@@ -55,13 +56,19 @@
...
+- [Escrow MeshJS](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src/escrow) - Escrow contract with full integration with MeshJS, offering code example and explanation.
+- [Marketplace MeshJS](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src/marketplace) - Cardano native token marketplace contract with full integration with MeshJS, offering code example and explanation.
+- [Payment Splitter MeshJS](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src/coupon-bond-guaranteed) - Payment splitter contract with full integration with MeshJS, offering code example and explanation.
+- [Simple Swap MeshJS](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src/swap) - Simple swap contract for token exchange with full integration with MeshJS, offering code example and explanation.
+- [Vesting Contract MeshJS](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src/vesting) - Vesting contract for token exchange with full integration with MeshJS, offering code example and explanation.
```
Comment (KtorZ) [inline README.md]:
```suggestion
- [MeshJS smart contracts collection](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src) - A series of smart contracts  with full integration with MeshJS, offering code examples and explanations
```

File: README.md [hunk 3]
```diff
@@ -55,13 +56,19 @@
...
 - [From Aiken to frontend deployment](https://meshjs.dev/guides/aiken) - A end-to-end guide from writing a Aiken smart contract, to creating transactions, to deploying a frontend to interact with it. See also [demo](https://aiken-next-ts-template.vercel.app/) and [repo](https://github.com/MeshJS/aiken-next-ts-template).
 - [Cardano Capture The Flag](https://github.com/vacuumlabs/cardano-ctf) - A game where Cardano developers and enthusiasts can try to exploit purposely vulnerable smart contracts and learn about the most common security issues and how to prevent them.
 - [Cardano Recorded Mint](https://github.com/keyan-m/cardano-recorded-mint) - A proof-of-concept for a minting script with historical record of all mints, and on-chain guarantee of uniqueness.
-- [Gimbalabs Aiken Essentials Module](https://plutuspbl.io/modules/303/slts) - Learn Aiken from scratch and develop your skills with hands-on exercises and mini-projects. 
+- [Gimbalabs Aiken Essentials Module](https://plutuspbl.io/modules/303/slts) - Learn Aiken from scratch and develop your skills with hands-on exercises and mini-projects.
```
Comment (KtorZ) [inline README.md:66]: The website isn't really working for me? None of the tutorial are accessible.

Comment (HinsonSIDAN) [reply README.md:66]: I update this line, think just formatter deleting last space. For the link, my best info is that it keep updating, maybe now is in middle of something.

Comment (KtorZ) [reply README.md:66]: Perhaps we should add it later, once it stabilizes?

Comment (HinsonSIDAN) [reply README.md:66]: Oh typo, actually i "didnt" update this line. I can pass the message to James for this, Im not in the loop for this so not the person to make decision

Comment (KtorZ) [reply README.md:66]: Ho. I see. My bad 🤦

**Review by KtorZ — APPROVED** [review]

**Pull request merged**